Dockerfile

```diff
@@ -32,7 +33,7 @@ RUN Rscript -e 'install.packages("civis")'
 # rocker/verse includes "fst" which has an AGPL license
 RUN Rscript -e 'remove.packages("fst")'
 
-ENV VERSION=4.0.4 \
```

Comment:
These were intended to be versions of the image, in line with releases in GitHub. They hadn't been updated with the last release. It doesn't seem like we really need to keep these variables around, but I went ahead and updated them.